cpp/kiss_icp/3rdparty/eigen/eigen.patch

```diff
@@ -0,0 +1,74 @@
+commit cf82186416d04ea5df2a397d8fe09dc78d40ca65
```

Comment:
Does this patch apply :thinking: ? I mean due the commit msg

Reply:
I tried locally and it worked:

![image](https://github.com/PRBonn/kiss-icp/assets/38326482/4b9a7bc2-0a63-4a85-bafd-c4aac969b061)

Reply:
all good np